src/Blazored.Video/BlazoredVideo.razor.Properties.cs

```diff
@@ -0,0 +1,215 @@
+﻿using System;
+using System.Runtime.CompilerServices;
+using Blazored.Video.Support;
+using Microsoft.JSInterop;
+
+namespace Blazored.Video
+{
+	public partial class BlazoredVideo
+	{
+		public bool Autoplay
```

Comment:
I'm not sure it makes sense to provide this as a settable property - does it have any effect?
It would be nice to have an example in the sample app(s) to show it working if it does.

Reply:
It loads the Video as soon as the source is loaded, if the source changes, if set to true this property should automatically start the playback. 

> Sets or returns whether the audio/video should start playing as soon as it is loaded